openapi/templates/README.mustache

```diff
@@ -56,7 +56,7 @@ namespace MyProject
         static void Main(string[] args)
         {
             Configuration config = new Configuration();
-            config.DefaultHeaders = new Dictionary<string, string>{{ "Accept", "application/vnd.mx.api.v1+json" }};
+            config.DefaultHeaders = new Dictionary<string, string>{{=<% %>=}}{{<%={{ }}=%> "Accept", "application/vnd.mx.api.v1+json" }};
```

Comment:
Is this the intended change? This doesn't look right as it has erb tags.

Reply:
Yeah, it is odd. There isn't a straightforward way to escape the `{{` in mustache unfortunately.

What it's doing is temporarily switching the delimiter to erb tags, inserting the "escaped" double curl braces, and then switching the delimiter back to double curly braces.

I followed the mustache guide [here](https://mustache.github.io/mustache.5.html) under the "Set delimiter" section to get it working.